modules/thermal_hydraulics/doc/content/source/materials/ADWallFrictionChengMaterial.md

```diff
@@ -0,0 +1,43 @@
+# ADWallFrictionChengMaterial
+
+The material computes the friction factor using the Cheng-Todreas correlation.
+
+Equations (9.105a), (9.105b), (9.109a), and (9.109b), and tables (9.5a) and (9.5b) from [!cite](todreas2021nuclear) are used.
+
+The Cheng-Todreas correlation allow us to calculate the friction factor for each subchannel type: interior, edge, or corner. The correlation is a function of the Pitch-to-Diameter ratio $(P/D)$ for interior channels, or the Wall-to-Diamter ratio $(W/D)$ for edge and corner subchannels. [fig-sub] shows the three subchannel types for a square and hexagonal rod bundles, where $P$ is the pitch between the fuel rods, $W$ is the distance between the fuel rod centerline and the bundle wall, and $D$ is the rod diameter.
+
+!media thermal_hydraulics/misc/subchannel_types.png
+       style=width:95%;float:center;
+       caption=Different subchannel types for a square and hexagonal array rod bundles.
+       id=fig-sub
```

Comment:
Nothing is displayed for some reason

Reply:
Yeap, I don't know how to put this image in the right place, it is in the Master branch of large_media, but it should be in the branch that next is using